src/stan-users-guide/parallelization.Rmd

```diff
@@ -213,20 +207,20 @@ functions {
 }
 ```
 
-And the likelihood statement in the model can now be written:
+The likelihood statement in the model can now be written:
 
 ```
 target += partial_sum(1, N, y, x, beta); // Sum terms 1 to N of the likelihood
 ```
 
 In this example, `y` was chosen to be sliced over because there
 is one term in the summation per value of `y`. Technically `x` would  have
-worked as well. Use whatever conceptually makes the most sense.
-
-Because `x` is a shared argument, it is subset accordingly with `start:end`.
-
-With this function, reduce summation can be used to automatically parallelize the
-likelihood:
+worked as well. Use whatever conceptually makes the most
+sense for a given model, e.g. slice over independent terms like
+conditionally independent observations or groups of observations as in
+hierarchical models. Because `x` is a shared argument, it is subset
```

Comment:
"is sliced" instead of "is subset"

Reply:
I would leave it here as subsetting which is the process *users* do to their data to get the chunk they need while slicing is what Stan-math does to the slicing argument.